packages/react-core/package.json

```diff
@@ -46,16 +46,17 @@
     "url": "https://github.com/knocklabs/javascript/issues"
   },
   "peerDependencies": {
-    "react": "^16.11.0 || ^17.0.0 || ^18.0.0"
+    "react": "^16.11.0 || ^17.0.0 || ^18.0.0 || ^19.0.0"
```

Comment:
Are there any issues with backwards compatibility to e.g. `16.11.0` with the changes we've made like Zustand? Here are the peer deps for zustand `5.0.3`

```
  "peerDependencies": {
    "@types/react": ">=18.0.0",
    "immer": ">=9.0.6",
    "react": ">=18.0.0",
    "use-sync-external-store": ">=1.2.0"
  },
  ```

Reply:
Aghhhh great call. Let me look deeper into this...

Reply:
I downgraded to zustand 4.5.6 to preserve React 16 support 😓